scripts/deploy_doc.sh

```diff
@@ -13,7 +13,7 @@ then
   TRAVIS_TAG="develop"
 fi
 
-docker run -it --rm -v "$PWD":/src -w /src -u "$(id -u "${USER}"):$(id -g "${USER}")" --env TRAVIS_TAG g0lden/mkdocs make site
+docker run -it --rm -v "$PWD":/src -w /src -u "$(id -u "${USER}"):$(id -g "${USER}")" --env TRAVIS_TAG g0lden/mkdocs make site || exit
```

Comment:
if you want travis to fail the build here, add non-zero exit code, i.e. `exit -1`

Reply:
I wanted exit from current script file. To avoid deleting current version of documentation in case build error